pkg/runtime/runtime.go

```diff
@@ -663,19 +668,30 @@ func (a *DaprRuntime) subscribeTopic(parentCtx context.Context, name string, top
 
 	ctx, cancel := context.WithCancel(parentCtx)
 	policy := a.resiliency.ComponentInboundPolicy(ctx, name, resiliency.Pubsub)
-	err := a.pubSubs[name].component.Subscribe(ctx, pubsub.SubscribeRequest{
+	var err error
+	routeMetadata := route.metadata
+	if routeMetadata[BulkSubscribe] == "true" {
```

Comment:
Consider using `utils.IsTruthy`

Reply:
https://github.com/dapr/dapr/pull/5285 addresses this.